### ISSUE TITLE
Staff of Protection

### ISSUE THREAD

**Issue by gtcoker** [RustyMods/AlmanacClasses#6]
When I cast the buffs from your spell lists and then cast the staff of protection, the game crashes.

**Comment by RustyMods** [comment]
My logs ?

**Comment by gtcoker** [comment]
How can I get the log? When I use the abilities from your system and then cast Staff of Protection, it instantly crashes, and nothing shows in the Unity Log as it force closes at the same time instantly. Is it captured somewhere else?

**Comment by RustyMods** [comment]
Staff of protection is the vanilla shield staff ?

**Comment by gtcoker** [comment]
Yes, it's the base game staff.  So if I cast it first, it will work. But if I cast your buffs first from the Almanac and then cast the protection staff, it crashes.

**Comment by gtcoker** [comment]
I did some debugging and found the spell that it breaks with. More specifically, when you cast RogueReflect or have the Rogue Reflect buff up and then cast Staff of Protection buff it crashes the game. The rest of my spells work fine.

**Comment by RustyMods** [comment]
just tested, game isn't crashing for me hmm
